tests/Unit/EmogrifierTest.php

```diff
@@ -1083,6 +1084,23 @@ public function emogrifyFiltersUnneededCssThings($css, $markerNotExpectedInHtml)
         static::assertNotContains($markerNotExpectedInHtml, $result);
     }
 
+    /**
+     * @test
+     *
+     * @param string $css
+     *
+     * @dataProvider unneededCssThingsDataProvider
+     */
+    public function emogrifyUnaffectedByUnneededCssThings($css)
```

Comment:
Please let's rename the test to be more specific. What does "unaffected" mean?

(And please add a verb so that the test name is a (possible pidgin) sentence.)

Reply:
Have renamed the method.